src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -872,6 +877,18 @@ public static function findByOptions($options)
                 ->getTracks()
             );
     }
+    /**
+     * This method allows direct access to iTonesPlaylists, without all the weird stuff in search api.$_COOKIE
+     *
+     * @param int itonesplaylistid The id of the itones playlist we want to get tracks for.
+    */
+    public static function itonesPlaylist($itonesplaylistid = null) {
```

Comment:
Hmm, Wasn't really in the mood for all the stuff regarding adding another API endpoint that way :( This was supposed to be quick, and I would finish API move during summer :(